src/modules/idle_inhibitor.cpp

```diff
@@ -23,6 +24,17 @@ waybar::modules::IdleInhibitor::IdleInhibitor(const std::string& id, const Bar&
   // Add this to the modules list
   waybar::modules::IdleInhibitor::modules.push_back(this);
 
+  thread_ = [this] {
```

Comment:
Check if timeout is defined before creating a useless thread

Reply:
Reworked to use a sigc::connection instead :)